dev_guide/builds.adoc

```diff
@@ -947,6 +947,15 @@ source secrets used. Access is granted with the following command:
 $ oc secrets link builder mysecret
 ----
 
+[NOTE]
+====
+As of {product-title} version 3.0.2, limiting secrets to only the service
```

Comment:
slightly clumsy wording, maybe just easier to say something along the lines of: if limitSecretReferences is false (default), this step is not required ?